.github/workflows/checks.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Checks
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  checks:
+    name: Consistency
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git clone repository
+        uses: actions/checkout@v2
+      - name: Prerequisite Deps
+        run: |
+          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata
+      - name: Run checksum checks on kickstart files
+        env:
+          LOCAL_ONLY: "true"
+        run: |
+          ./tests/installer/checksums.sh
+      - name: Web Dashboard pre-generated file consistency checks (dashboard.js)
+        run: |
+          cp web/gui/dashboard.js /tmp/dashboard.js
+          autoreconf -ivf
+          ./configure --enable-maintainer-mode
+          make dist
+          diff -sNrdu /tmp/dashboard.js web/gui/dashboard.js
```

Comment:
I had not thought of this originally, but these should probably be separate jobs, not separate steps of a single job. They're orthogonal to each other, and they run in parallel in Travis right now (jobs in Travis run in parallel, it's stages that are serialized).

Reply:
Yup okay I _thought_ about doing this but I was in split minds about it.

Reply:
Done